packages/qwik-city/buildtime/vite/dev-server.ts

```diff
@@ -194,6 +194,9 @@ export function ssrDevMiddleware(ctx: BuildContext, server: ViteDevServer) {
 
       next();
     } catch (e) {
+      if (res.writableEnded) {
```

Comment:
we've extended the `qwik` dev server middleware to handle errors as well instead of forwarding the to the vite error middleware. this because we are otherwise unable to ship the needed CSS to modify the overlay on SSR errors. The new error handler terminates the response with an `.end()` call. After that point it does no longer make sense to follow the `next(e)` path at all because an `header already sent` error would occur.